integration/tests/cook/util.py

```diff
@@ -467,3 +467,13 @@ def sleep_for_publish_interval(cook_url):
     progress_publish_interval_ms = get_in(cook_settings, 'progress', 'publish-interval-ms')
     wait_publish_interval_ms = min(3 * progress_publish_interval_ms, 20000)
     time.sleep(wait_publish_interval_ms / 1000.0)
+
+def output_progress_string(cook_url, percent, message):
```

Comment:
Maybe just `progress_line`? The "output" is kind of confusing

Reply:
Done.